src/ruby/yast/logger.rb

```diff
@@ -1,6 +1,10 @@
 require "yastx"
 require "yast/builtins"
 
+# FIXME: a workaround for https://github.com/yast/yast-ruby-bindings/issues/133
+require "enc/encdb.so"
+require "enc/trans/transdb.so"
```

Comment:
I think that this workaround should be placed near ruby initializer: https://github.com/yast/yast-ruby-bindings/blob/master/src/binary/YRuby.cc#L75